reader.go

```diff
@@ -120,11 +120,14 @@ func NewReader(r io.Reader) *Reader {
 	}
 }
 
-// Read reads each line of the ACH file and defines which parser to use based on the first character
-// of each line. It also enforces ACH formatting rules and returns the appropriate error if issues are found.
+// Read reads each line in the underlying io.Reader and returns a File and any errors encountered.
 //
-// A parsed file may not be valid and callers should ensure the file is validate with Validate()
-// and tabulate the file with Create(). Invalid files may be rejected by other Financial Institutions or ACH tools.
+// Read enforces ACH formatting rules and the first character of each line determines which parser is used.
+//
+// The returned File may not be valid. Callers should tabulate the File with File.Create followed by
```

Comment:
Following the comment styles from the Go Standard library, I'm not using `()` when referencing functions. Is it fine if we start conforming to this convention?

Reply:
I think we should, looks like we already do in a lot of comments. @adamdecaf, what do you think?  I can go through and remove all the `()`s if we decide to go that route.

Reply:
Removing `()`'s is fine with me.